R/robustbayesianmetaanalysis.R

```diff
@@ -1231,52 +1235,52 @@ RobustBayesianMetaAnalysis <- function(jaspResults, dataset, options, state = NU
     
   }
   
-  prior_overtitle <- gettext("Prior Distribution")
+  overtitlePrior <- gettext("Prior Distribution")
```

Comment:
overTitlePrior

Reply:
that would make it inconsistent with the JASP own's internal argument naming, e.g. `overtitle` argument in `addColumnInfo` function

`modelsSummary$addColumnInfo(name = "priorMu",    title = gettext("Effect Size"),      type = "string",  overtitle = overtitlePrior)`

Reply:
Sure keep it that way then.